File: whale-data/whale-data-api/src/main/java/me/whale/data/api/view/UserVo.java
package me.whale.data.api.view;

public class UserVo {
    private String userName;

    public UserVo(String userName) {
        this.userName = userName;
    }

    public String getUserName() {
        return userName;
    }

    public void setUserName(String userName) {
        this.userName = userName;
    }
}

File: whale-web/whale-site/src/main/kotlin/me/whale/web/view/UserVo.kt
package me.whale.web.view

import jakarta.validation.constraints.NotBlank
import jakarta.validation.constraints.Size


data class UserVo(
    @field:NotBlank val userNo: String,
    @field:NotBlank val userName: String,
    val gender: String,
    val birthday: String,
    @field:Size(min = 8, max = 255) val password: String
)